docs/topics/classes.md

```diff
@@ -1,245 +1,692 @@
+To create an object of a class, use the class name followed by parentheses (`()`), similar to calling a function:
 
 ```kotlin
-val invoice = Invoice()
+// Creates an object of the 'Person' class
+val person1 = Person()
+```
 
-val customer = Customer("Joe Smith")
+In Kotlin, you can create objects:
+
+* **Without arguments** (`()`): creates an object using a constructor with default values.
+* **With arguments** (`("value")`): creates an object by passing specific values to the constructor.
+
+As you can see, you can assign the created object to a mutable (`var`) or immutable (`val`) [variable](basic-syntax.md#variables):
+
+```kotlin
+// Creates an object by passing a specific value and assigns it to a mutable variable
+var person2 = Person("Joe")
+
+// Creates an object using the default constructor's value and assigns it to an immutable variable
+val person1 = Person()
 ```
 
-> Kotlin does not have a `new` keyword.
->
-{style="note"}
+You can create objects wherever you need them, inside the `main()` function, within other functions, or as part of a class. 
 
-The process of creating instances of nested, inner, and anonymous inner classes is described in [Nested classes](nested-classes.md).
+When creating objects inside the `main()` function, when the program runs, execution starts from `main()`. 
+Here is where you instantiate objects to interact with them. When creating objects of a class, you're invoking constructors 
+(for example, `Person()` or `Person("Joe")`) directly inside the `main()` function.
 
-## Class members
+Let's look at an example of how object creation works.
 
-Classes can contain:
+The following code defines a `Person` class with a property for storing a name. 
+When an object of the class is created, an initialization block runs to print a message. The example demonstrates 
+creating an object with both the default constructor's value and a specific value:
 
-* [Constructors and initializer blocks](classes.md#constructors)
-* [Functions](functions.md)
-* [Properties](properties.md)
-* [Nested and inner classes](nested-classes.md)
-* [Object declarations](object-declarations.md)
+```kotlin
+// Class header with primary constructor that initializes 'name' with a default value
+class Person(val name: String = "Sebastian")
+
+fun main() {
+    // Creates an object using the default constructor's value
+    val person1 = Person()
+    println("Person object created: ${person1.name}")
+    // Person object created: Sebastian
+
+    // Creates an object by passing a specific value
+    val person2 = Person("Joe")
+    println("Person object created: ${person2.name}")
+    // Person object created: Joe
+}
+```
+{kotlin-runnable="true"}
+
+> In Kotlin, unlike other object-oriented programming languages like Java, 
+> there is no need for the `new` keyword when creating objects of a class.
+>
+{style="note"}
+
+Additionally, you can create objects inside another function and call that function from `main()`.
```

Comment:
I think we've already said that before, and repeating the same fact again feels weird.